pipelines/common/src/main/java/com/google/fhir/analytics/AvroConversionUtil.java

```diff
@@ -146,14 +146,8 @@ static synchronized AvroConversionUtil getInstance(
   public synchronized FhirContext getFhirContext() throws ProfileMapperException {
```

Comment:
I think this is still needed, right? Maybe I can remove it after your PR is merged?

Reply:
Since the piece of code which throws the `ProfileMapperException` has been removed in this PR, I thought we could also remove the `throws` clause from the method definition. With my PR, this is getting changed to `ProfileException` only.